lib/libversion.py

```diff
@@ -360,4 +399,4 @@ def match(self, version):
             else:
                 if not op(vi, evi):
                     matches = False
-        return matches
```

Comment:
Add a blank line at the end of the file (as discussed in PEP8) to avoid the ⛔ below in the GitHub file viewer.